gix-ref/src/store/file/loose/iter.rs

```diff
@@ -93,7 +87,7 @@ impl file::Store {
     /// Return an iterator over all loose references that start with the given `prefix`.
     ///
     /// Otherwise it's similar to [`loose_iter()`][file::Store::loose_iter()].
-    pub fn loose_iter_prefixed(&self, prefix: &Path) -> std::io::Result<LooseThenPacked<'_, '_>> {
+    pub fn loose_iter_prefixed(&self, prefix: Cow<'_, BStr>) -> std::io::Result<LooseThenPacked<'_, '_>> {
```

Comment:
This should be `&BStr` if it was `&Path` before.
It could also be `impl Into<…>` if that makes it more convenient to call, even for the test-suite.

Reply:
I put `impl Into<Cow<...>>` in most places. Not sure if some should just be `&BStr` or not.